dev-docs/source/Testing/SANSGUI/ISISSANSGUITests.rst

```diff
@@ -171,8 +171,9 @@ Processing
 #. Take note of the name of the reduced workspace with ``merged`` in the title.
 #. Make a copy of the row you just processed using the ``Copy`` and ``Paste`` buttons above the runs table.
 #. Change the ``Output Name`` of the new row to something like ``bgsub_test``.
-#. In the ``Options`` column, enter ``BackgroundWorkspace=<WS_NAME>, ScaleFactor=0.9`` (replacing <WS_NAME> with the
-   name of the merged workspace you took note of before).
+#. Check the ``Scaled Background Subtraction`` checkbox.
+#. In the ``BackgroundWorkspace`` column, enter the name of the merged workspace you took note of before.
+#. In the ``ScaleFactor`` column, enter ``0.9``.
 #. Select this new row and click ``Process Selected``.
```

Comment:
I remembered when testing that there is a known behaviour where changing information in the batch table causes manually changed GUI settings to revert to the user file value. Is there something earlier in these test instructions that means this won't be an issue here, or do we need to set the reduction mode only after we've finished editing the batch table?

Reply:
Good point! I'd forgotten about that feature. I'll add that in

Reply:
Thinking about it, you'll probably need it both at the beginning and later on unfortunately because it will need to be set to merged before running the first reduction to create the background workspace. Then, after entering the background subtraction settings, it will change back to all so it will need to be changed again. Perhaps one approach could be to set everything up in the batch table, switch to merged reduction and then use the button that processes all the rows in sequence rather than doing them one at a time? We should be able to provide the name that will be output for the merged background workspace in the testing instructions.

Reply:
I think I'm going to alter the instructions to just change the user file. This is the last step of the runs tab testing, so it should be safe to change it at this point without destroying all the future tests.

Reply:
I think the only consideration is if people change it then we would need to make sure it's correct next time they start the manual testing instructions again, in case they haven't refreshed their training data (I know that I don't re-download it very frequently).